### ISSUE TITLE
zign's a component but not listed or mentioned on stups.io home page

### ISSUE THREAD

**Issue by lasomethingsomething** [zalando-stups/documentation#81]
[No sign](https://stups.io/) of Zign.

[But it's here](https://stups.readthedocs.io/en/latest/components/index.html).

cc @gargravarr

**Comment by harti2006** [comment]
Isn't zign just a CLI tool for the proprietary (not open-source) employee token service?

**Comment by hjacobs** [comment]
@harti2006 yes, that's true, but it's also used by all CLI tools (as a library) and it can generate service tokens via the Python tokens library (which works with Open Source Plan B).
